modules/eventing/pages/eventing-rbac.adoc

```diff
@@ -29,7 +29,18 @@ A bucket.scope combination is used for identifying functions belonging to the sa
 Only the "Eventing Full Admin" role and also the "Full Admin" role can set the bucket.scope to  *+*+.+*+*; all other Eventing non-privileged users need to define a *Function Scope* for their Eventing functions that references an existing resource of bucket.scope. 
 This provides role based isolation of Eventing functions between non-privileged users
 
-Typically you should set Function Scope to the bucket.scope that holds the collection that is the source of your mutations to your Eventing Function.  This best practice ensures that you _do not_  inadvertently cause an Eventing Function to undeploy by removing a *Function Scope* pointing to a resource that is not required for the function to run.
+// tag::rbac-change-warning[]
+[CAUTION]
+====
+Changing the access role (i.e., by revoking write permissions) could impact deployed eventing functions that have been assigned to this role. It is possible that the function may be undeployed; in this case, redeploy the function with a correctly assigned role to allow access.
+====
+// end::rbac-change-warning[]
+
+Typically, you should set Function Scope to the bucket.scope
```

Comment:
this section here is oddly ventilated (e.g. line breaks not just at full stop, or even commas, but at logical chunks)?
I don't hate it, but it's maybe a little arbitrary, so more susceptible to weird diffs if next writer disagrees with the chunking.

Reply:
Yeah, see what you mean. I'll take another look.